atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraKeyValueServiceConfig.java

```diff
@@ -163,6 +165,11 @@ default String getKeyspaceOrThrow() {
      */
     Optional<SslConfiguration> sslConfiguration();
 
+    @Value.Default
+    default CqlClientFactory cqlClientFactory() {
```

Comment:
This was one of my concerns, does it make sense to have it here or put everything in the `CqlCapableConfig`. However it can so I'll move it, it will require some changes.

Reply:
Decided to leave it in the top level, the code later is simplified.